code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -1260,21 +1260,24 @@
 		to_chat(caller, SPAN_DANGER("You cannot translate (muted)."))
 		return
 
-	caller.create_typing_indicator()
+	var/list/heard = get_mobs_in_view(7, caller)
```

Comment:
Did you test this? it might make deaf people still hear it since its before the message send check